_example/echo/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/arl/statsviz"
+	"github.com/labstack/echo/v4"
+)
+
+func main() {
+	// Echo instance
+	e := echo.New()
+
+	// Statsviz
+	// Add monitor Go application runtime statistics (GC, MemStats, etc.)
+	// Create a new http ServeMux
+	mux := http.NewServeMux()
+	// Register route
+	_ = statsviz.Register(mux)
+
+	// Use echo WrapHandler to wrap statsviz ServeMux as echo HandleFunc
+	e.GET("/debug/statsviz/", echo.WrapHandler(mux))
+	// Server static content for statsviz UI
+	e.GET("/debug/statsviz/*", echo.WrapHandler(mux))
+
+	// Start server
+	e.Logger.Fatal(e.Start(":1323"))
```

Comment:
Can we use port 8080 as other examples?
```suggestion
	e.Logger.Fatal(e.Start(":8080"))
```